crates/core_simd/src/vector.rs

```diff
@@ -12,7 +12,39 @@ pub(crate) mod ptr;
 use crate::simd::intrinsics;
 use crate::simd::{LaneCount, Mask, MaskElement, SupportedLaneCount};
 
-/// A SIMD vector of `LANES` elements of type `T`.
+/// A SIMD vector of `LANES` elements of type `T`. `Simd<T, N>` has the same shape as [`[T; N]`](array), but operates like `T`.
+/// This type is commonly known by names like `f32x4` or `Vec4` in many programming languages.
+///
+/// Two vectors of the same type and length will, by convention, support the binary operations (+, *, etc.) that `T` does.
```

Comment:
I know what you mean here, but binary might be confusing wrt bitwise operations, too.  I would maybe just say arithmetic operations.

Reply:
:upside_down_face: Mmm. I am going to just say "operators", thinking about it, because some of the operators are **not** arithmetical...